e2e/capacity/staking.test.ts

```diff
@@ -189,13 +187,13 @@ describe("Capacity Staking Tests", function () {
 
     describe("when staking and targeting an InvalidTarget", async function () {
         it("fails to stake", async function () {
-            const maxMsaId = (await ExtrinsicHelper.getCurrentMsaIdentifierMaximum()).toNumber();
-
             const stakeAmount = 10n * CENTS;
-            const stakeKeys = await createAndFundKeypair(fundingSource, stakeAmount + 1n, "StakeKeys");
+            const stakeKeys = await createAndFundKeypair(fundingSource, accountBalance, "StakeKeys");
+
+            const [notProviderMsaId] = await createMsa(fundingSource);
 
-            const failStakeObj = ExtrinsicHelper.stake(stakeKeys, maxMsaId + 1, stakeAmount);
```

Comment:
Because this test is running in parallel, it cannot just use maxMsaId + 1 as other msas are being created and might be providers.